snap/config/security-gateway-api/kong.conf.in

```diff
@@ -0,0 +1,498 @@
+# -----------------------
```

Comment:
Where did this file originate?  Can it be copied instead of duplicated?

Reply:
I modified it from the default kong config, basically only thing modified is the prefix option, which I supposed I could modify with sed like I already do in the install hook.

Reply:
This is now modified from the default kong config file with sed.